dd-sdk-android/src/test/kotlin/com/datadog/android/rum/internal/domain/scope/RumActionScopeTest.kt

```diff
@@ -688,6 +684,46 @@ internal class RumActionScopeTest {
         assertThat(result).isNull()
     }
 
+    @Test
+    fun `𝕄 send event with user extra attributes 𝕎 handleEvent(any)`(
+        @IntForgery(1) count: Int,
+        forge: Forge
+    ) {
+        // Given
+        testedScope.viewTreeChangeCount = count
+        Thread.sleep(RumActionScope.ACTION_INACTIVITY_MS)
+
+        // When
+        val result = testedScope.handleEvent(mockEvent(), mockWriter)
+
+        // Then
+        argumentCaptor<RumEvent> {
+            verify(mockWriter).write(capture())
+            assertThat(lastValue)
+                .hasUserExtraAttributes(fakeUserInfo.extraInfo)
```

Comment:
Actually I think you can add this check in all test methods, the same way we assert the `hasUserInfo` in all tests.